modules/contracts/apps/subscription/examples/schema.rs

```diff
@@ -0,0 +1,34 @@
+use abstract_subscription::contract::SubscriptionApp;
+use abstract_subscription::msg::{StateResponse, SubscriberResponse, SubscriptionFeeResponse};
+use abstract_subscription::state::SubscriptionConfig;
+use cosmwasm_schema::{export_schema, export_schema_with_title, remove_schemas, schema_for};
+use cw_asset::{AssetInfo, AssetInfoUnchecked};
+use std::env::current_dir;
+use std::fs::create_dir_all;
+
+fn main() {
+    let mut out_dir = current_dir().unwrap();
```

Comment:
Are these required like this?
I.e. do we need those extra state and config exports?